controllers/badge.js

```diff
@@ -20,6 +20,20 @@ const getBadges = async (req, res) => {
   }
 }
 
+const getUserBadges = async (req, res) => {
+  try {
+    const allUserBadges = await badgeQuery.fetchUserBadges(req.params.username)
+    return res.json({
```

Comment:
Need to handle the case when the username given is invalid

Reply:
Okay, I'll make the changes.